src/screens/UserPortal/UserScreen/UserScreen.tsx

```diff
@@ -130,7 +130,10 @@ const UserScreen = (): JSX.Element => {
       >
         <div className="d-flex justify-content-between align-items-center">
           <div style={{ flex: 1 }}>
-            <h1>{titleKey !== 'home' ? t('title') : ''}</h1>
+            {/* Before  */}
+            {/* <h1>{titleKey !== 'home' ? t('title') : ''}</h1>   */}
+            {/* After  */}
+            <h1>{titleKey !== 'home' ? t('title') : 'Posts'}</h1>
```

Comment:
I  did this 
![image](https://github.com/user-attachments/assets/216289f5-f168-43f9-9ef6-81745aa4d84f)
because by doing this +<h1>{titleKey !== 'home' ? t('title') : t('posts.title')}</h1>
the result was 
![image](https://github.com/user-attachments/assets/63209dd7-f373-4617-8e4e-cb46c9ef4d2f)

Reply:
it is not helping and what you told me to do is giving me this result. 
![image](https://github.com/user-attachments/assets/6b9813b8-8036-4e7d-b291-a0d5366fa4ca)